filesystems/tests/common.py

```diff
@@ -882,6 +1126,105 @@ def test_create_existing_directory_from_link_with_parents(self):
             os.strerror(errno.EEXIST) + ": " + str(link),
         )
 
+    # TODO: cover links to all of:
```

Comment:
These TODOs we'll handle in a follow-up?

Reply:
I haven't carefully compared before and after but I think I improved the 'coverage'.  So my plan was that that would be a future endeavor.  Though I wasn't explicitly volunteering to do it in the near term.  I understand if that affects the PR acceptance.